feature-libs/pickup-in-store/components/container/my-preferred-store/my-preferred-store.component.ts

```diff
@@ -22,31 +29,44 @@ import { ICON_TYPE } from '@spartacus/storefront';
 import { Observable } from 'rxjs';
 import { filter, map, switchMap, take, tap } from 'rxjs/operators';
 
+const getDirectionsName = 'Get Directions';
```

Comment:
Preferably these would be translatable. But lets leave that out of the scope of this fix.

Can we use `GET_DIRECTION_NAME` and `CHANGE_STORE_NAME` names for these variables so its clear that they should be final?